src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/types.js

```diff
@@ -5,6 +5,7 @@ import type {AttributeMap} from '../../services/Router';
 
 export type ViewProps = {
     children?: (?Object) => Element<*> | null,
+    isRootView ?: boolean,
```

Comment:
```suggestion
    isRootView?: boolean,
```

Reply:
maybe we could even make this non-nullable?
```suggestion
    isRootView: boolean,
```

but is okay for me to leave it nullable if it makes things easier

Reply:
If you insist, I can change it to non-nullable, but then we would have to adjust every test or every view with default props